lib/mix/tasks/phx.gen.channel.ex

```diff
@@ -55,6 +56,7 @@ defmodule Mix.Tasks.Phx.Gen.Channel do
     mix phx.gen.channel expects just the module name:
 
         mix phx.gen.channel Room
+
```

Comment:
I had other PRs upcoming that include a new line on error messages in case there is indented content at the end. Is that OK?